src/schema/rules/files/raw/dwi.yaml

```diff
@@ -40,10 +40,14 @@ sbref:
     chunk: optional
 
 # Common scanner-generated derivatives need raw names
-isotropic:
+ScannerDerivatives:
   suffixes:
     - ADC
-    - TRACE
+    - FA
+    - S0map
+    - colFA
+    - expADC
+    - trace
```

Comment:
Looking at the differences between this and the anatomical S0map:

https://github.com/bids-standard/bids-specification/blob/817454ba05d70893287e2d619230f1fc01b26b8d/src/schema/rules/files/raw/anat.yaml#L34-L66

* No `task` - makes sense
* No `ceagent` - makes sense as long as plain DWI does not permit it
* Has `direction` - Is that still applicable for these files?

Reply:
RE `S0` more generally, I originally revised based on what I read in the description, where it exclusively referenced the projected TE=0 signal intensity in fMRI. If it's also used in anatomical imaging, maybe its description should be revised accordingly; see suggestion below.

RE `direction`: in a pipeline context one would expect data from all phase encoding directions to have been combined prior to estimation of such a map. But as a scanner-generated derivative (on Siemens at least), if an imaging experiment involves multiple phase encoding directions, they are run as separate sequence protocols, and so if this scanner-generated derivative is activated there will be one generated per phase encoding direction. So persisting there as optional is right I think.